js/adapt-assessmentArticleModel.js

```diff
@@ -198,6 +198,16 @@ const AssessmentModel = {
   _setupBanks() {
     const assessmentConfig = this.getConfig();
     const bankSplits = assessmentConfig._banks._split.split(',');
+
+    this.findDescendantModels('block')
+      .filter(block => block.get('_isAvailable'))
+      .filter(block => block.findDescendantModels('question').length > 0)).forEach(block => {
```

Comment:
Currently getting an error while compiling with line 204 expecting a semi colon. 

Could we not combine the two .filters and have 
.filter(block => (block.get('_isAvailable') && block.findDescendantModels('question').length > 0))

Reply:
Thanks Joe, I missed that

Reply:
Apologies, I put an extra closing parentheses which has been copied across. Line 204.

Reply:
Ah, I should have been more careful! Thanks for the heads up